manifest.json

```diff
@@ -2028,6 +2081,16 @@
                 }
             },
             "versions": {
+                "1.1.2": {
+                    "releaseUrl": "https://github.com/kawaiiamber/GifImporter/releases/tag/1.1.2",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/kawaiiamber/GifImporter/releases/download/1.1.2/GifImporter.dll",
+                            "filename": "GifImporter.dll",
+                            "sha256": "cc16edc29f849cb717ff4cff350c96a04a76852a96d556588caab6eae698e2d1"
+                        }
+                    ]
+                },
```

Comment:
- Uses it's own instance of WebClient, bypassing Neos' internal possible web request checks (or other mods too). Also without too much investigation into the logic, a quick glance would suggest that it does web requests for completely unrelated image imports too.
- Uses the `nml_mods` folder to save temporary files instead of using a proper temp files folder.

Reply:
WebClient thing looks to bypass using any checks provided by neos or other mods. It requests headers for the file to check the type first which still makes that request.

Reply:
Would need to actually verify if neos catches it and if we deem that a big enough issue to decline it otherwise as the earlier version of this is already part of the manifest.